src/main/kotlin/org/rust/cargo/runconfig/CargoTestRunState.kt

```diff
@@ -29,7 +29,8 @@ class CargoTestRunState(
 
     private val cargoTestPatch: CargoPatch = { commandLine ->
         val rustcVer = cargoProject?.rustcInfo?.version
-        commandLine.copy(additionalArguments = patchArgs(commandLine, rustcVer))
+        // TODO: always pass `withSudo` when `com.intellij.execution.process.ElevationService` supports error stream redirection
+        commandLine.copy(additionalArguments = patchArgs(commandLine, rustcVer), withSudo = false)
```

Comment:
Let's add link to the corresponding issue in our tracker (of course, it requires creating such issue)